src/relay/WalletSDKRelay.ts

```diff
@@ -122,6 +123,19 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
       this.eventListener,
     );
 
+    this.subscriptions.add(
+      this.connection.sessionConfig$.subscribe({
+        next: sessionConfig => {
+          this.onSessionConfigChanged(sessionConfig);
```

Comment:
Do you need to optionally call this function in the case it's not implemented?

Reply:
This function will never not be implemented. It will either have a nop-empty implementation or one overridden by extending the class.